### DISCUSSION TITLE
可以用telegram的命令删除图片吗

### DISCUSSION THREAD

**Discussion by Rebellce** [y-young/nazurin#106] [Q&A] [answered]
图片保存在telegram里手动删了好像没用，再次收藏提示已经在数据库里了，但实际上已经在相册里已经没有保存这张图了=/

**Comment by y-young** [comment]
很遗憾目前不行，需要手动在数据库里删除，但如果在数据库里有就说明所有 Storage 都已经保存成功了。

**Reply by Rebellce** [reply]
哦哦，好的谢谢。具体要怎么手动删除呢，数据库是用的默认的。项目是在Ubuntu上用的docker部署的。

**Reply by y-young** [reply, marked as answer]
在 Docker 里用默认数据库记得持久化，不然容器删除后数据就没了。默认的就是个 JSON 文件，只要在 `data/pixiv.json` 里找到对应 Artwork ID 的 key 删掉就行了。
